src/login/73-seat-late.rules.m4

```diff
@@ -13,7 +13,8 @@ ENV{ID_SEAT}=="", ENV{ID_AUTOSEAT}=="1", ENV{ID_FOR_SEAT}!="", ENV{ID_SEAT}="sea
 ENV{ID_SEAT}=="", IMPORT{parent}="ID_SEAT"
 
 ENV{ID_SEAT}!="", TAG+="$env{ID_SEAT}"
-
-TAG=="uaccess", ENV{MAJOR}!="", RUN{builtin}+="uaccess"
+m4_ifdef(`HAVE_ACL',``
+TAG=="uaccess", ENV{MAJOR}!="", RUN{builtin}+="uaccess"''
```

Comment:
hmm, i am a bit confused by the quoting here?
```
… `` … '' …
```

Why double quote here? We don't do that at the other places and I can't see why that'd be needed? But maybe I am missing something regarding m4 syntax here? Can you enlighten me?

Reply:
Quoting here is needed to escape the commas in the udev rules. Single quotes would also work in this case but double quotes ensure the argument is treated as a literal string and nothing inside is expanded if it is also a macro. See the [m4 quoting rules](https://www.gnu.org/savannah-checkouts/gnu/m4/manual/m4-1.4.18/html_node/Quoting-Arguments.html#Quoting-Arguments).